config/features.yaml

```diff
@@ -0,0 +1,45 @@
+# Config file for the feature extraction analyzer.
+# A feature extraction definition looks like this:
+# name:
+#       # Define either a query_string or query_dsl.
+#       query_string: *
+#       query_dsl:
+#       # Mandatory fields.
+#       attribute:
+#       store_as:
+#       re:
+#       # Optional fields.
+#       emojis: []
+#       tags: []
+#       create_view: False
+#       aggregate: False
+#
+# Each definition needs to define either a query_string or a query_dsl.
+#
+# The fields tags and emojis are optional.
+#
+# The field store_as defines the name of the attribute the feature is
+# stored as.
+#
+# The create_view is an optional boolean that determines whether a view
+# should be created if there are hits.
+#
+# The aggregate is an optional boolean that determines if we want to
+# create an aggregation of the results and store it (ATM this does
+# nothing, but once aggregations are supported it will).
+#
+# The feature extraction works in the way that the query is run, and
+# the regular expression is run against the attribute to extract a value.
+# The first value extracted is then stored inside the "store_as" attribute.
+# If there are emojis or tags defined they are also applied to that event.
+# In the end, if a view is supposed to be created a view searching for
+# the added tag is added (only if there are results).
+# ------------------------------------------------------------------------
+
+browser_usernames:
+        query_string: 'source_short:"WEBHIST"'
+        attribute: 'message'
+        store_as: 'username'
```

Comment:
Isn't this an email address that is extracted? A username is a bit misleading I think.

Reply:
done